### DISCUSSION TITLE
Pytorch 2.0 reduced ops specs

### DISCUSSION THREAD

**Discussion by nlgranger** [artyom-beilis/pytorch_dlprim#45] [Announcements]
Hi and congrats for the hard work.

The release of PyTorch 2.0 mentioned [a reduced opset to implement](https://pytorch.org/get-started/pytorch-2.0/#primtorch-stable-primitive-operators) in order to add a new backend. Did you look into it and would it make your work easier?

**Comment by artyom-beilis** [comment]
Thanks. I'm defiantly  going to look into it!
